src/app/scripts/cac/map/cac-map-control.js

```diff
@@ -101,7 +107,7 @@ CAC.Map.Control = (function ($, L, _) {
             deferred.fail(function(){return 'ERROR(' + error.code + '): ' + error.message; });
         };
 
-        if ('geolocation' in navigator) {
+        if (navigator.geolocation) {
```

Comment:
This looks cleaner, but it causes problems in various browsers. See the "First things first" section here: http://themarklee.com/2013/10/27/super-simple-geolocation/